_includes/author-profile.html

```diff
@@ -1,28 +1,32 @@
 {% assign author = page.author | default: page.authors[0] | default: site.author %}
 {% assign author = site.data.authors[author] | default: author %}
 
-<div itemscope itemtype="https://schema.org/Person">
+<div itemscope itemtype="https://schema.org/Person" class="h-card">
+
+  <a class="u-url hidden" rel="me" href="{{ '/' | relative_url }}">
```

Comment:
Can we remove this hidden markup since `u-url` is being applied to the `author.home` link.
If `u-url` is required and the thought was to add the hidden markup because there is no guarantee someone adds `author.home` to their config.

Perhaps we can do a similar thing to the post title. In this case make the author name a link... use `author.home` first fallback to the site's root URL if the home link hasn't been configured.

```liquid
<h3 class="author__name" itemprop="name" class="p-name">{{ author.name }}</h3>
  <a class="u-url" rel="me" href="{{ author.home | default: '/' | relative_url }}">{{ author.name }}</a>
</h3>
```